config/initializers/vendor/pp_sql.rb

```diff
@@ -0,0 +1,17 @@
+if Rails.env.development? && defined?(Rails::Console)
+  require 'pp_sql'
+
+  if ENV['RAILS_PP_SQL_CONSOLE_LOGGING']
```

Comment:
Maybe 'TW_<something>' ?

Reply:
I thought of that too after I pushed of course, thanks for catching it.